doc/usage/restructuredtext/basics.rst

```diff
@@ -258,8 +258,8 @@ target formats (HTML, LaTeX) have a limited supported nesting depth.
 Field Lists
 -----------
 
-Field lists (:duref:`ref <field-lists>`) are sequences of fields marked up like
-this::
+Field lists (:ref:`docutils ref <field-lists>`) are sequences of fields
+marked up like this::
```

Comment:
Could this be done as a separate PR? I don't think there's a dependency here, right?